src/sentry/workflow_engine/models/workflow.py

```diff
@@ -10,13 +10,28 @@
 from sentry.constants import ObjectStatus
 from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model, sane_repr
 from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+from sentry.db.models.manager.base import BaseManager
 from sentry.models.owner_base import OwnerModel
 from sentry.workflow_engine.models.data_condition import DataCondition, is_slow_condition
 from sentry.workflow_engine.types import WorkflowEventData
 
 from .json_config import JSONConfigBase
 
 
+class WorkflowManager(BaseManager["Workflow"]):
```

Comment:
might be nice to have some individual tests for the managers to ensure it's working as expected.